### ISSUE TITLE
Example APP using streamlit-authenticator

### ISSUE THREAD

**Issue by Rhyzhang** [mkhorasani/Streamlit-Authenticator#24]
Hi @mkhorasani,

Thank you for this amazing component!

It would be helpful if you created an example app using streamlit-authenticator so it is easier to quickstart using this component. I am having some issues using the component.

Best,
Ryan

**Comment by mkhorasani** [comment]
Hi @Rhyzhang,

You are most welcome! The readme file is effectively a sample app. Aternatively, you can take a look at this [video](https://www.youtube.com/watch?v=JoFGrSRj4X4&t=1s) which covers the process for older versions of this package (<v0.2.1). Can you also please elaborate on what issues you are having?

Thanks.

**Comment by Rhyzhang** [comment]
Hi @mkhorasani,

I built an app using stauth v0.1.5 and was very happy with how it turned out. However, I am interested in implementing v0.2.1 because of its new features. I was having issues with the login where even though I typed in the correct username and password the authenticator did not authenticate me. It stayed at None instead of True. Unfortunately, I have reverted my changes to using v0.1.5 so I do not have the error anymore. I decided to build out the rest of the app on whatever I have now. I might have accidentally made a mistake in my attempt to quickly implement v0.2.1. That being said, I would still love to use v0.2.1 of streamlit authenticator. I will reattempt to use v0.2.1 soon.

Many thanks!
